deploy/docker-compose.yml

```diff
@@ -3,6 +3,9 @@ services:
   tink-server:
     build: ../cmd/tink-server
     restart: unless-stopped
+    command:
+      - --tls-cert=/certs/${FACILITY:-onprem}/bundle.pem
+      - --tls-key=/certs/${FACILITY:-onprem}/server-key.pem
```

Comment:
This was previously needed for the purposes of the old vagrant test, I'm not sure if this compose file is actually used anywhere now.